### DISCUSSION TITLE
Meeting agenda 19th April 2023

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#61] [Meeting Agenda]
Hi All,

This is our draft agenda for our next TSC meeting. Items to discuss include:

- Review of actions from [previous meeting](https://github.com/finos/technical-steering-committee/blob/master/meetings/2023-03-15.md)
  - Mao has created a discussion thread on the [project activation process and potential TSC involvement](https://github.com/finos/technical-steering-committee/discussions/58)
  - A discussion thread was created to [solicit hackathon ideas](https://github.com/finos/technical-steering-committee/discussions/59)
- Continue the discussion regarding TSC involvement in project activation, there is [1 pending activation](https://github.com/finos/community/issues/230)
- An update on the TSC governance progress, there have been some proposed changes which need consideration 
- Discuss proposed usage of [meeting minutes template](https://github.com/finos/technical-steering-committee/pull/56)

**Comment by mindthegab** [comment]
Shall we talk about an in person meeting at Member meeting / Open House in London?
